react/react1/week1/Todo/src/components/TodoList/TodoList.jsx

```diff
@@ -0,0 +1,19 @@
+import TodoListItem from "../TodoListItem/TodoListItem"
+import './TodoList.css'
+
```

Comment:
I can see that you've specified which props you want in your components. This is perfectly fine, and will help you make more stable components, especially when working in teams.